rest/reference.go

```diff
@@ -104,10 +111,7 @@ func (c *ReferenceClient) GetMarketStatus(ctx context.Context, options ...models
 //
 // This method returns an iterator that should be used to access the results via this pattern:
 //
-//	iter, err := c.ListSplits(context.TODO(), params, opts...)
-//	if err != nil {
-//		return err
-//	 }
```

Comment:
If we are removing the error handling, we should add back the correct way of checking for errors.